test/pgtest-mz/portals.pt

```diff
@@ -0,0 +1,40 @@
+# In Materialize we eagerly commint implicit transactions after Execute
+# messages, causing portals to be destroyed.
+send
+Parse {"query": "VALUES (1), (2)"}
+Bind {"portal": "c"}
+Execute {"portal": "c", "max_rows": 1}
+Execute {"portal": "c", "max_rows": 1}
+Sync
+----
+
+until
+ReadyForQuery
+----
+ParseComplete
+BindComplete
+DataRow {"fields":["1"]}
+PortalSuspended
+ErrorResponse {"fields":[{"typ":"S","value":"ERROR"},{"typ":"C","value":"34000"},{"typ":"M","value":"portal \"c\" does not exist"}]}
+ReadyForQuery {"status":"I"}
+
+# Verify that portals (cursors) are destroyed on Execute.
+send
+Parse {"query": "VALUES (1), (2)"}
+Bind {"portal": "c"}
+Execute {"portal": "c", "max_rows": 1}
+Sync
+Query {"query": "FETCH c"}
+----
+
+until err_field_typs=M
```

Comment:
Files in this directory should never need `err_field_typs`. That directive is there only when we need to have exactly the same output as postgres, but our error codes or messages don't match exactly.

Reply:
Fixed